container-search/src/main/resources/configdefinitions/qr-start.def

```diff
@@ -14,6 +14,9 @@ jvm.gcopts string default="-XX:+UseG1GC -XX:MaxTenuringThreshold=15 -XX:NewRatio
 ## Heap size (in megabytes) for the Java VM
 jvm.heapsize int default=1536 restart
 
+## Min heapsize for the Java VM
+jvm.minheapsize int default=1536 restart
```

Comment:
Could we call it `minHeapsize` for readability?

Reply:
Fixed